packages/medusa/src/api-v2/store/regions/query-config.ts

```diff
@@ -6,22 +6,15 @@ export const defaultStoreRegionFields = [
   "updated_at",
   "deleted_at",
   "metadata",
-  "countries.id",
```

Comment:
thought: Think we still need the countries in the storefront

Reply:
@olivermrbl I was just thinking it might be better to not fetch it by default, WDYT?